x/pki/client/cli/query_child_certificates.go

```diff
@@ -6,58 +6,26 @@ import (
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 	"github.com/zigbee-alliance/distributed-compliance-ledger/x/pki/types"
 )
 
-func CmdListChildCertificates() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "list-child-certificates",
-		Short: "list all ChildCertificates",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx := client.GetClientContextFromCmd(cmd)
-
-			pageReq, err := client.ReadPageRequest(cmd.Flags())
-			if err != nil {
-				return err
-			}
-
-			queryClient := types.NewQueryClient(clientCtx)
-
-			params := &types.QueryAllChildCertificatesRequest{
-				Pagination: pageReq,
-			}
-
-			res, err := queryClient.ChildCertificatesAll(context.Background(), params)
-			if err != nil {
-				return err
-			}
-
-			return clientCtx.PrintProto(res)
-		},
-	}
-
-	flags.AddPaginationFlagsToCmd(cmd, cmd.Use)
-	flags.AddQueryFlagsToCmd(cmd)
-
-	return cmd
-}
-
 func CmdShowChildCertificates() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "show-child-certificates [issuer] [authority-key-id]",
-		Short: "shows a ChildCertificates",
-		Args:  cobra.ExactArgs(2),
+		Use:   "all-child-x509-certs",
```

Comment:
as  understand we don't provide a cli to get all child certificates for any given certificates. It's ok but the current single-item command naming contradicts with other modules (auth, validator) since `all-...` is used there for list queries only.

Reply:
it has been discussed, no issue here